iFramed/iFramed.js

```diff
@@ -0,0 +1,48 @@
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, regexp: true, indent: 4, maxerr: 50 */
```

Comment:
Rename this dir to `browser/` and the file to `iframe-browser.js`, so it will be `browser/iframe-browser.js`